crates/runtime/src/dataaccelerator/cayenne.rs

```diff
@@ -226,6 +226,14 @@ impl Default for CayenneAccelerator {
     }
 }
 
+fn parse_usize(acceleration: &Acceleration, key: &str, default: usize) -> usize {
+    acceleration
+        .params
+        .get(key)
+        .and_then(|v| v.parse::<usize>().ok())
```

Comment:
nit: Don't think you need `.ok()` because you can `.unwrap_or()` a Result.


```suggestion
        .and_then(|v| v.parse::<usize>())
```

Reply:
`.and_then()` must return an `Option<U>`